packages/vuetify/src/directives/touch/__tests__/touch.spec.ts

```diff
@@ -2,95 +2,88 @@
 import Touch from '../'
 
 // Libraries
-import Vue from 'vue'
+import { defineComponent, withDirectives, h, createApp, DirectiveBinding, nextTick } from 'vue'
 
 // Utilities
-import {
-  mount,
-  Wrapper,
-} from '@vue/test-utils'
 import { touch } from '../../../../test'
 
 describe('touch.ts', () => {
-  let mountFunction: (value?: object) => Wrapper<Vue>
+  let mountFunction: (value?: object) => HTMLElement
+  const el = document.createElement('div')
 
   beforeEach(() => {
     mountFunction = (value = {}) => {
-      return mount(Vue.component('test', {
-        directives: { Touch },
-        render: h => h('div', {
-          directives: [{
-            name: 'touch',
-            value,
-          }],
-        }),
-      }))
+      const Test = defineComponent(() => () => withDirectives(h('div', { class: 'test' }), [ [ Touch, value ] ]))
+      const app = createApp()
+
+      app.mount(Test, el)
+      return el
```

Comment:
Could return `el.querySelector('.test')` here
Also not sure why it's not just defined before `describe`? In `beforeEach` always the same value is assigned, it seems pointless